src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -32,6 +32,45 @@ g.test('format')
     t.doCreateRenderPipelineTest(isAsync, !!info.depth || !!info.stencil, descriptor);
   });
 
+g.test('depthCompare')
+  .desc(`The depthCompare in depthStencilState is optional for stencil-only formats.`)
+  .params(u => u.combine('isAsync', [false, true]).combine('format', kTextureFormats))
```

Comment:
nit: we could test only depth stencil formats, the validation that a format must be a depth stencil format is already done in the test above.

Reply:
This would remove the need for a bunch of the `!!` too.

Reply:
Also can just be a subcase param.

Reply:
Done. Thanks for the suggestion!